### PR TITLE
bundleStoreWrapper added

### PR DESCRIPTION
**Pull request by Anze1m**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [x] Corner cases evaluated and handled
- [x] index.js exports updated
- [x] Automatic tests added/updated
- [x] Automatic tests pass
- [x] Sanity check passed
- [x] Self-review passed
- [x] Version updated

### CODE REVIEW

File: src/wrappers/bundle_store_wrapper.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+import ManagedContractWrapper from './managed_contract_wrapper';
+
+export default class BundleStoreWrapper extends ManagedContractWrapper {
+  get getContractName() {
+    return 'BundleStore';
```
Comment (spherefoundry) [inline src/wrappers/bundle_store_wrapper.js]:
```suggestion
    return 'bundleStore';
```
